webpack.config.js

```diff
@@ -26,6 +28,16 @@ const config = {
       filename: 'async/index.html',
       template: './demo/async/index.html',
     }),
+    new HtmlWebpackPlugin({
+      inject: false,
+      filename: 'paymentIntents/index.html',
+      template: './demo/paymentIntents/index.html',
+    }),
+    new webpack.DefinePlugin({
+      'process.env.STRIPE_SECRET_KEY': JSON.stringify(
+        process.env.STRIPE_SECRET_KEY
+      ),
+    }),
```

Comment:
I don't see any readme changes in this PR.

- It seems like we should at least add the new methods to the code block in the readme that starts `type FactoryProps = {`.
- Is it odd that the examples in the readme all use `createToken`, but this PR changes all demos to use `createPaymentMethod`? I wonder whether we should have a token demo, or a readme sample using cPM.

Reply:
Ah, I missed that all the demos switched over to cPM, will switch back. I wanted to move all the PI specific stuff into a demo.

Reply:
Also readme changes inc, still todo